pkg/sdk/context_functions.go

```diff
@@ -130,6 +159,21 @@ func (c *contextFunctions) CurrentUser(ctx context.Context) (string, error) {
 	return s.CurrentUser, nil
 }
 
+func (c *contextFunctions) Current(ctx context.Context) (*CurrentDetails, error) {
```

Comment:
okay, i see now. Is this really necessary though? just because it was done in a similar way in the old snowflake package, doesn't mean we need to do it again here. also there are more current functions than just what is listed here.

Reply:
There are multiple times when we want to fetch both account and region, for me it is better to fetch all at once and not shoot for every single param.